src/sequential_cursor.rs

```diff
@@ -72,7 +76,7 @@ impl<'s> std::io::Seek for SeqCursor<'s> {
                     .cursors
                     .iter()
                     .fold(0u64, |acc, cursor| acc + cursor.get_ref().len() as u64);
```

Comment:
Can `let total =` be collapsed into `self.len`?  It looks like it's just calculating that length of all the slices underlying the cursors on the fly, but we're already keeping track of that.

Reply:
That's unrelated to the fix presented, and I'd prefer to get this merged ASAP rather than pull in more change of lines that were not touched.

Reply:
@dralley I would not change this in current PR.